src/DynamoCoreWpf/ViewModels/Search/NodeSearchElementViewModel.cs

```diff
@@ -278,16 +279,22 @@ protected virtual void CreateAndConnectToPort(object parameter)
 
                 // Create a new node based on node creation name and connection ports
                 dynamoViewModel.ExecuteCommand(new DynamoModel.CreateAndConnectNodeCommand(id, initialNode.GUID,
-                    Model.CreationName, 0, Model.AutoCompletionNodeElementInfo.PortToConnect, adjustedX, 0, createAsDownStreamNode, false, true));
+                    Model.CreationName, 0, Model.AutoCompletionNodeElementInfo.PortToConnect, adjustedX, adjustedY, createAsDownStreamNode, false, true));
             }
             else
             {
                 // Placing the new node to the left of initial node
                 adjustedX -= initialNode.Width + 50;
 
+                // If the new node is a slider input node, adjust the position on X-axis to compensate for higher width of the slider node.
+                if (Model.CreationName.Contains("Slider")) 
```

Comment:
I think this will restrict this improvement to only those wide nodes that have `Slider` in its name, can we have a more generic approach here that may cover all wide nodes, such as getting the width of the incoming node and then calculating the `adjustedX`.

Reply:
The issue is during this function call, the new node is not yet created and we don't know the width of the node. I couldn't think of any other nodes that are so wide. Do you recall any?

Reply:
Ah..yes I remember the issue now, thanks.